Makefile

```diff
@@ -157,9 +155,20 @@ clean:
 	# crossbuild artifacts
 	rm -f build/mgmt-*
 
-test: bindata
+test: build
 	./test.sh
 
+# create all test targets for make tab completion (eg: make test-gofmt)
+test_suites=$(shell find test -maxdepth 1 -name test-* -exec basename {} .sh \;)
```

Comment:
is it more appropriate to use `find test/` (with trailing slash so it's more readable and explicit)
(same below)

Reply:
I agree, that was my first approach, but then you end up with double slashes in paths like:
```
~/.g/s/g/p/mgmt (testscripting|REBASE-i 1/2|✚17…2) $ find test/
test/
test//omv
test//omv/helloworld.yaml
test//omv/pkg1a.yaml
test//omv/pkg1b.yaml
```

Which technically doesn't matter at all i think, but causes me to go into correctness limbo :).

But i think we can favor for clearness in the Makefile.